slurmci.jl

```diff
@@ -36,9 +36,12 @@ function load_jobs(sha::String, tag::String)
     function create_test_job(sname, entry)
         cmd = `$(sname) $(entry["file"])`
         for arg in entry["args"]
-            push!(cmd.exec, arg)
+            arg_parts = split(arg)
+            for arg_part in arg_parts
+                push!(cmd.exec, arg_part)
+            end
```

Comment:
```suggestion
            append!(cmd.exec, Base.shell_split(arg))
```
`Base.shell_split` handles standard shell quoting